### PR TITLE
Update to 1.19

### PR DESCRIPTION
**Pull request by KdotJPG**
- `new TranslatableText` -> `Text.translatable`
- bump versions

Might want to create a 1.18 branch off the current `master` before merging this, so that it's convenient to make further changes to both versions.

### CODE REVIEW

File: gradle.properties [hunk 1]
```diff
@@ -3,14 +3,14 @@ org.gradle.jvmargs=-Xmx1G
...
 	# check these on https://fabricmc.net/versions.html
-	minecraft_version=1.18
-	yarn_mappings=1.18+build.1
-	loader_version=0.12.6
+	minecraft_version=1.19
+	yarn_mappings=1.19+build.1
+	loader_version=0.14.6
 
 # Mod Properties
 	mod_version = 1.0.1
```
Comment (Fictitious-Rotor) [inline gradle.properties:11]: Don't forget to increment this!

**Comment by Fictitious-Rotor** [comment]
Was literally about to make a pull request with almost exactly this code hahaha

File: gradle.properties (hunk 1 above)
Comment (KdotJPG) [reply gradle.properties]: Good idea. Made it ` 1.0.2-Fabric-1.19` in favor of more-sustainable versioning with multiple target Minecraft versions.

Comment (PotatoPresident) [reply gradle.properties]: > Good idea. Made it ` 1.0.2-Fabric-1.19` in favor of more-sustainable versioning with multiple target Minecraft versions.

It already appends the mc version in the build.gradle

**Comment by PotatoPresident** [comment]
Totally forgot about this PR until I went to see if this worked in 1.19.1 sorry! Just going to merge into master, don't foresee having any major additions/changes to backport,

**Pull request merged**